spec/lib/pdf_filler/nc_d400_pdf_spec.rb

```diff
@@ -43,36 +48,36 @@
           expect(pdf_fields['y_d400wf_fname1']).to eq 'North'
           expect(pdf_fields['y_d400wf_mi1']).to eq 'A'
           expect(pdf_fields['y_d400wf_lname1']).to eq 'Carolinianian'
-          expect(pdf_fields['y_d400wf_ssn1']).to eq '400000030'
-          expect(pdf_fields['y_d400wf_add']).to eq '123 Red Right Hand St Apt 1'
-          expect(pdf_fields['y_d400wf_apartment number']).to eq 'Apt 1'
-          expect(pdf_fields['y_d400wf_city']).to eq 'Raleigh'
+          expect(pdf_fields['y_d400wf_ssn1']).to eq '145004904'
+          expect(pdf_fields['y_d400wf_add']).to eq '7 Heavens Lane'
+          expect(pdf_fields['y_d400wf_apartment number']).to eq 'Apt 2'
+          expect(pdf_fields['y_d400wf_city']).to eq 'Cary'
           expect(pdf_fields['y_d400wf_state']).to eq 'NC'
-          expect(pdf_fields['y_d400wf_zip']).to eq '27513'
+          expect(pdf_fields['y_d400wf_zip']).to eq '27511'
 
           expect(pdf_fields['y_d400wf_fstat1']).to eq 'Yes'
           expect(pdf_fields['y_d400wf_fstat2']).to eq 'Off'
           expect(pdf_fields['y_d400wf_fstat3']).to eq 'Off'
           expect(pdf_fields['y_d400wf_fstat4']).to eq 'Off'
           expect(pdf_fields['y_d400wf_fstat5']).to eq 'Off'
 
-          expect(pdf_fields['y_d400wf_li6_good']).to eq '9000'
-          expect(pdf_fields['y_d400wf_li8_good']).to eq '9000'
+          expect(pdf_fields['y_d400wf_li6_good']).to eq '76902'
+          expect(pdf_fields['y_d400wf_li8_good']).to eq '76902'
           expect(pdf_fields['y_d400wf_ncstandarddeduction']).to eq 'Yes'
           expect(pdf_fields['y_d400wf_li11_page1_good']).to eq '12750'
 
           expect(pdf_fields['y_d400wf_lname2_PG2']).to eq 'Carolinian'
-          expect(pdf_fields['y_d400wf_li20a_pg2_good']).to eq '15'
-          expect(pdf_fields['y_d400wf_li23_pg2_good']).to eq '15'
-          expect(pdf_fields['y_d400wf_li25_pg2_good']).to eq '15'
-          expect(pdf_fields['y_d400wf_dayphone']).to eq '9845559876'
+          expect(pdf_fields['y_d400wf_li20a_pg2_good']).to eq '4394'
+          expect(pdf_fields['y_d400wf_li23_pg2_good']).to eq '4394'
+          expect(pdf_fields['y_d400wf_li25_pg2_good']).to eq '4394'
```

Comment:
notes for self: line 20a= sum of all state income tax amount, which this fixture only has one of `<StateIncomeTaxAmt>4394</StateIncomeTaxAmt>`
line 23 =line_or_zero(:NCD400_LINE_20A) + line_or_zero(:NCD400_LINE_20B)
line 25 = same as line 23 (line 24 OOS)